pkg/printer/printer_test.go

```diff
@@ -23,17 +23,17 @@ func TestPrintWarnings(t *testing.T) {
 		},
 	},
 		{
-			Title:           "Failed with yaml validation",
+			Title:           "/datree/datree/internal/fixtures/kube/yaml-validation-error.yaml\n",
```

Comment:
is this means that the current printings are changing?

Reply:
No, the prints will stay the same, the tests didn't show how the function should act